addon/initializers/embedded.ts

```diff
@@ -33,12 +33,32 @@ function normalizeConfig(userConfig:GivenConfig):ObjectConfig {
 
   if (configIsBoolean(userConfig)) {
     let embeddedConfig = { delegateStart: userConfig, config: {} }
-    deprecate('The `embedded` config property MUST be undefined or an an object', false, { id: 'bad-object-config', until: '1.0.0' })
+    deprecate(
+      'The `embedded` config property MUST be undefined or an an object',
+      false, {
+        id: 'ember-cli-embedded.bad-object-config',
+        until: 'not defined',
+        for: 'ember-cli-embedded',
+        since: {
+          enabled: '1.0.0'
+        }
+      }
+    )
     return embeddedConfig
   }
 
   if (userConfig.delegateStart === undefined) {
-    deprecate('The config MUST contain a `delegateStart` property. Assuming `true` for backward compatibility. The config must now be defined in a `config` property', false, { id: 'bad-object-config', until: '1.0.0' })
+    deprecate(
+      'The config MUST contain a `delegateStart` property. Assuming `true` for backward compatibility. The config must now be defined in a `config` property',
+      false, {
+        id: 'ember-cli-embedded.bad-object-config',
+        until: 'not defined',
+        for: 'ember-cli-embedded',
+        since: {
+          enabled: '1.0.0'
```

Comment:
See my comment https://github.com/peopledoc/ember-cli-embedded/pull/117/files#r692914148 .

```suggestion
          enabled: '0.5.0'
```